### PR TITLE
Add reportFlow param to the broken site report pixel

### PR DESCRIPTION
**Pull request by lmac012**
Task/Issue URL: https://app.asana.com/0/488551667048375/1205957959864913/f

### Description

Add the reportFlow parameter to the broken site report pixel

#### Menu flow
- [x] Navigate to the site breakage report screen through menu -> Report Broken Site
- [x] Send report
- [x] Verify that `"reportFlow"="menu"` param is included in the `epbf` pixel.

#### Dashboard flow
- [x] Navigate to the site breakage report screen through Privacy Dashboard -> "Website not working?"
- [x] Send report
- [x] Verify that `"reportFlow"="dashboard"` param is included in the `epbf` pixel.

### CODE REVIEW

**Comment by lmac012** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #3881** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/3881?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/duckduckgo/Android/3881?utm_source=stack-comment).

File: app/src/main/java/com/duckduckgo/app/brokensite/BrokenSiteActivity.kt [hunk 1]
```diff
@@ -86,6 +88,7 @@ class BrokenSiteActivity : DuckDuckGoActivity() {
         val errorCodes = intent.getStringArrayExtra(ERROR_CODES).orEmpty()
         val httpErrorCodes = intent.getStringExtra(HTTP_ERROR_CODES).orEmpty()
         val isDesktopMode = intent.getBooleanExtra(IS_DESKTOP_MODE, false)
+        val reportFlow = intent.getSerializableExtra<ReportFlow>(REPORT_FLOW)!!
```
Comment (joshliebe) [inline app/src/main/java/com/duckduckgo/app/brokensite/BrokenSiteActivity.kt:91]: We generally avoid `!!` in code (It's fine in tests), is there a way to handle the null case more gracefully?

**Review by joshliebe — CHANGES_REQUESTED** [review]
Other than the `!!`, LGTM and works as expected 👍
Once that's addressed, I'll approve 🙂

**Review by joshliebe — APPROVED** [review]
Re-tested, works as expected. Good job 👍

**Pull request merged**